app/models/wiki.rb

```diff
@@ -1,7 +1,7 @@
 class Wiki < ActiveRecord::Base
   belongs_to :user
   has_many :collaborations
-  has_many :users, through: :collaborations
+  has_many :collaborators, through: :collaborations, source: :user
 
```

Comment:
Same here, `Collaboration` only knows about users, so this line defines an alias, `collaborators`, for a collaboration's users attribute.